hbase-common/src/main/java/org/apache/omid/tools/hbase/X509Util.java

```diff
@@ -0,0 +1,288 @@
+/*
```

Comment:
I don't think that the TLS functionality belongs to hbase-common module.
We don't actually use it when connecting to HBase.

IMO this belongs in the omid-common module instead.